### ISSUE TITLE
regarding learning rate for canonical gaussians and deformation model; and how this relates to training stability/time(until convergence))

### ISSUE THREAD

**Issue by Jinjing98** [ingra14m/Deformable-3D-Gaussians#82]
Hi, thanks for your great work!

1. When compare deformable-3d-gs to the hust 4D gs method, i observe much slower training time(quality still outperform), could you please give some hints (both of you adopt MLP, in addtion hust 4D gs adopt hexplane)?
I try to figure out the reason from learning rate perspective, while encounter a second issue:

2. in the paper, you refer adopting the same lr setting as original gs paper for the gaussians model, while it seems you hard code the scene_extent to be a constant 5. isn't this inconsistent with the paper? or this not matters much?
![image](https://github.com/user-attachments/assets/f8d5e623-a9fd-4742-864c-ce7c62050435)

https://github.com/ingra14m/Deformable-3D-Gaussians/blob/ac423728df780ee57a82025a6d7c4c7312fdf9bb/scene/gaussian_model.py#L88
https://github.com/ingra14m/Deformable-3D-Gaussians/blob/ac423728df780ee57a82025a6d7c4c7312fdf9bb/scene/gaussian_model.py#L117

Also could you breif the intuiation for a different learning rate design in the deformation model as introduced in the paper?

**Comment by JianxiangL** [comment]
Do you have any further understanding of this? I'm quite curious as well.
